src/status_im/ui/screens/browser/views.cljs

```diff
@@ -17,7 +17,7 @@
 (views/defview toolbar-content-dapp [contact-identity]
   (views/letsubs [contact [:contact-by-identity contact-identity]]
     [react/view styles/toolbar-content-dapp
-     [chat-icon.screen/dapp-icon-browser contact]
+     [chat-icon.screen/dapp-icon-browser contact 36]
```

Comment:
Extract as a constant?

Reply:
too many honors this  number

Reply:
it's just style

Reply:
I've seen it twice in your changes thus the question.

Reply:
yeah but it's still a difficult question, are they related and should be changed simultaneously or not

Reply:
:+1: